apps/condo/domains/billing/utils/testSchema/mixins/organization.js

```diff
@@ -21,6 +22,11 @@ const OrganizationTestMixin = {
         await createTestOrganizationEmployee(this.clients.admin, this.organization, this.clients.employee[type].user, role)
     },
 
+    async updateOrganization (updateInput) {
+        const [updatedOrganization] = await updateTestOrganization(this.clients.admin, this.organization.id, updateInput)
```

Comment:
Why everything is created / updated from admin perspective? Its not close to production environment, where there's no admins at all

Reply:
@dkoviazin ^

Reply:
Because, there is a limitation on editing some fields for organization. For example, tin
And we already have tests for updating Organization - so there is no need to test it again in utils

Reply:
Yep, but mixin pattern is bad because of that

Reply:
in my head mixins is added on top of actor, so actor should call all by himself

Reply:
we need an utils to fast create Organization, Property, etc....
we use them only for that
then we test new mutations on the old way